src/main/resources/db/migration/V6_1__add_duplicate_field.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE person_oppgave ADD COLUMN duplikat_referanse_uuid CHAR(36) DEFAULT NULL;
```

Comment:
Ble litt usikker - duplikat er vel bare et fenomen for sykmeldingene, så burde dette feltet heller høre hjemme i `sykmelding`-tabellen?

Reply:
Jeg tenkte at det er dette feltet er det som skal brukes i syfomodiaperson til å vise at en personoppgave er potensiell duplikat (dvs at det skal være med i dto'en som returneres til syfomodiaperson). Så jeg synes det enkleste er å ha dette som en kolonne i personoppgave-tabellen.

Reply:
Hmm, blir det ikke veldig tettkobling mellom de to tjenestene hvis vi lar en konsument av et API legge føringer for domenemodellen i databasen? 

Ville en join med sykmelding-tabellen gi samme resultat når man henter opp personoppgaver for en person, eller er det ytelse-issues med det?

Reply:
Nå må ikke dette nødvendigvis være noe som bare brukes for personoppgaver laget fra sykmeldinger: vi bruker jo referanseuuid også for andre typer oppgaver (de kan referere til dialogmeldinger, dialogmøter eller oppfølgingsplaner feks). Så da kan jo potensielt dette duplikatfeltet også brukes for andre typer oppgaver.

Reply:
> Det vil bare være en forekomst i sykmelding-tabellen for den første sykmeldingen, ikke for den som duplikat-matchet.

Ah, okey, jeg trodde vi lagret ned sykmeldinger i `sykmelding`-tabell, og så bare opprettet oppgave i `personoppgave` når det ikke var duplikat. Og at da ville man potensielt ha _n_ sykmeldinger lagret med en referanse til samme personoppgave-uuid. Men ser jo nå at hvis vi skal lage oppgaver for alle sykmeldinger uansett så blir ikke det riktig logikk.

Foreløpig er det altså bare å telle og vise duplikater, og ingen filtrering av oppgaver? Mulig jeg tenkte litt flere steg fremover her.

Reply:
Ja, det er riktig. Og så tenkte jeg å tilpasse syfomodiaperson slik at veileder gjøres oppmerksom på at en sykmeldingsoppgave er duplikat av en tidligere oppgave (derfor er duplikatReferanseUuid med i PersonOppgaveVeileder-dto'en).